ScreeningEncounterForm.html

```diff
@@ -0,0 +1,1209 @@
+    {"label": "Endebes District Hospital", "val": "162"},
+    {"label": "Kapsara District Hospital", "val": "132"},
+    {"label": "Kiminini Health Centre", "val": "310"},
+    {"label": "Kitale District Hospital", "val": "11"},
+    {"label": "Kwanza Health Centre", "val": "320"},
+    {"label": "Matunda Health Centre", "val": "324"},
+    {"label": "Mt. Elgon Dispensary", "val": "9"},
+    {"label": "Saboti District Hospital", "val": "82"},
+    {"label": "Bidii Health Centre", "val": "330"},
+    {"label": "St Fredahs Mission Hosp", "val": "345"},
+    {"label": "Chepngoror Dispensary", "val": "232"},
+    {"label": "Kipkabus Dispensary", "val": "223"},
+    {"label": "Burnt Forest Sub District Hosp", "val": "4"},
+    {"label": "Pioneer Health Centre", "val": "70"},
+    {"label": "West Clinic Health Centre", "val": "336"},
+    {"label": "Soy Dispensary", "val": "73"},
+    {"label": "SOS Dispensary", "val": "145"},
+    {"label": "Mois Bridge Health centre", "val": "71"},
+    {"label": "Langas Dispensary", "val": "341"},
+    {"label": "Kapsoya Health Centre", "val": "343"},
+    {"label": "Ziwa Sub District Hosp", "val": "28"},
+    {"label": "Magemo", "val": "239"},
+    {"label": "Sinoko", "val": "75"},
+    {"label": "Mitukuyu", "val": "241"},
+    {"label": "Marinda", "val": "242"},
+    {"label": "Nang'eni", "val": "243"},
+    {"label": "Chulaimbo HC", "val": "7"},
+    {"label": "Riat Dispensary", "val": "103"},
+    {"label": "Sunga Dispensary", "val": "104"},
+    {"label": "Siriba Dispensary", "val": "105"}
+    ];
+
+    var cdm_follow_up_facility_names= [
+    {"label": "Mosoriot Health Centre", "val": "2"},
+    {"label": "Birbiriet Dispensary", "val": "111"},
+    {"label": "Itigo Dispensary", "val": "112"},
+    {"label": "Chepsaita Dispensary", "val": "60"},
+    {"label": "Cheramei  Dispensary", "val": "116"},
+    {"label": "Nyenyilel Dispensary", "val": "128"},
+    {"label": "Ososrogai Dispensary", "val": "137"},
+    {"label": "Turbo Health Centre", "val": "3"},
+    {"label": "Ngechek Dispensary", "val": "115"},
+    {"label": "Lelemokwo Health Centre", "val": "113"},
+    {"label": "Chepterit Mission Hospital", "val": "133"},
+    {"label": " Murgo Hills Dispensary", "val": "136"},
+    {"label": "Murgusi  Dispensary", "val": "117"},
+    {"label": "Sambut  Health Centre", "val": "127"},
+    {"label": "Sosiani Health Centre", "val": "129"},
+    {"label": "Turbo Health Care Centre", "val": "155"},
+    {"label": "Kokwet Dispensary", "val": "114"},
+    {"label": "Mogoget Dispensary", "val": "110"},
+    {"label": "Chepkemel Dispensary", "val": "122"},
+    {"label": "Cheplaskei Health Centre", "val": "118"},
+    {"label": "Sugoi B Dispensary", "val": "121"},
+    {"label": "Sugoi A Dispensary", "val": "120"},
+    {"label": "Leseru/Trinity Hospital Dispensary", "val": ""},
+    {"label": "Sigot Dispensary", "val": "119"},
+    ];
+
+    var locationName = [
+	{"label": "Malaba Dispensary", "val": "91"},
```

Comment:
Eventually we will download location dynamically, right? So we could potentially remove this references (including providers and other location related info).